developer/styleguide.md

```diff
@@ -54,66 +55,111 @@ In our Markdown documentation, we use [remark-lint](https://github.com/wooorm/re
 
 Pull request branches are evaluated using [BitHound](https://www.bithound.io/github/reactioncommerce/reaction) for insecure dependencies and code quality.
 
-- must pass a Linting check and *no errors* are accepted.
-- must pass a Dependency check and no priority packages are accepted.
-- must pass `reaction test`
+-   must pass a Linting check and _no errors_ are accepted.
+-   must pass a Dependency check and no priority packages are accepted.
+-   must pass `reaction test`
 
 In many cases, documentation updates can be required as well.
 
 Pull requests are submitted to a peer code review process before acceptance.
 
-### File Naming Conventions
+### Naming Conventions
 
-In general we use hyphens (-) and camelCase for folder names, and camelCase alone for file names. Underscores are not to be used for file or folder names unless expressly required.  Be aware that not all operating systems are case sensitive, so it's not ok to have two files named the same with differing case.
+In general we use hyphens (-) for folder names. Underscores are not to be used for file or folder names unless expressly required.  Be aware that not all operating systems are case sensitive, so it's not ok to have two files named the same with differing case. To prevent this, we recommend not using camelCasing when naming folders or files.
```

Comment:
Does using hyphens in filenames cause issues in meteor? I recall having issues where hyphenated files in import directories were not being loaded